py_bentoctl/other_helpers.py

```diff
@@ -2,6 +2,11 @@
 import pathlib
 import shutil
 import sys
+import json
+import subprocess
+from typing import Optional
+import uuid
+from datetime import datetime, timezone
```

Comment:
reorder imports - `import` first, then `from` + both sublists in alphabetical order